scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<ol><li>Could you run black formatting on this?</li><li>Consistent use of osp (for os.path) - Sometimes osp.join is used, other times os.path.join is used.</li><li>Best practice for the <code>open</code> function in python is to do something like <code>with open(path) as f: do something</code></li><li>Use f-strings to be consistent with other print statements in the codebase, e.g. `print(f"Re-run the inference, saved in path {result_file_path}")</li><li>Maybe the save print statement could be simply phrased as "Saved results to {result_file_path}"</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>